README.md

```diff
@@ -9,19 +9,21 @@ The home of Zowe API Mediation Layer
 
```

Comment:
We should include a general description of who is the target audience of this README and what this README includes.

Reply:
The readme is being rewritten, you can expect another PR with it